tests/disaster-recovery/sc_arbiter/test_noobaa_in_stretch.py

```diff
@@ -0,0 +1,90 @@
+import pytest
+import logging
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework import config
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.helpers.helpers import modify_deployment_replica_count
+from ocs_ci.ocs.resources.pod import (
+    wait_for_pods_by_label_count,
+    get_pods_having_label,
+    Pod,
+    get_pod_node,
+)
+from ocs_ci.framework.pytest_customization.marks import (
+    bugzilla,
+    polarion_id,
+    tier3,
+    red_squad,
+    mcg,
+)
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def setup_nb_endpoint():
+    nb_endpoint_dep = OCP(
+        kind="Deployment",
+        resource_name="noobaa-endpoint",
+        namespace=config.ENV_DATA["cluster_namespace"],
+    )
+    nb_endpoint_dep_data = nb_endpoint_dep.get()
+    if nb_endpoint_dep_data["spec"]["replicas"] != 2:
+        old_replicas = nb_endpoint_dep_data["spec"]["replicas"]
+        modify_deployment_replica_count(nb_endpoint_dep.resource_name, replica_count=2)
+
+    def teardown():
+        modify_deployment_replica_count(
+            nb_endpoint_dep.resource_name, replica_count=old_replicas
+        )
+
+    return nb_endpoint_dep_data
+
+
```

Comment:
If the test is specific to the arbiter, please add a 'stretchcluster' marker so that the test can be ignored for other deployment types

Reply:
done